src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard_de.properties

```diff
@@ -136,8 +136,7 @@ doclet.Preview_API_Checkbox_Toggle_All=Alle umschalten
 doclet.Preview_JEP_URL=https://openjdk.org/jeps/{0}
 doclet.Preview_Label=Vorschau
 doclet.Preview_Mark=PREVIEW
-doclet.Preview_Notes=Hinweise zur Vorschau-API
```

Comment:
The changes in this group of localized files should not be making it into JDK25 as the associated change to the original file made it into 26 and was not backported to 25.